src/sinks/vector/mod.rs

```diff
@@ -0,0 +1,96 @@
+pub mod v1;
+pub mod v2;
```

Comment:
This solution is based on our Lua v1/v2 implementation. I still have to do some additional testing to make sure this is backward compatible (it should be), and is working as intended.